demos/dox/sequence/segment.cpp

```diff
@@ -94,5 +94,27 @@ int main()
     std::cout << preB << "\n";  // => "XXx"
 //![explicit segment]
 
+    // This section below is used to remove unused typedef warnings but is not
+    // shown anywhere within the docs. Only the fragments above are displayed.
+    TInfix2  _inf2;
+    TInfix3  _inf3;
+    TInfix4  _inf4;
+    TInfix5  _inf5;
+    TPrefix2 _pref2;
+    TInfix6  _inf6;
+    TInfix7  _inf7;
+    TInfix8  _inf8;
+    TSuffix2 _suf2;
+
+    ignoreUnusedVariableWarning(_inf2);
+    ignoreUnusedVariableWarning(_inf3);
+    ignoreUnusedVariableWarning(_inf4);
+    ignoreUnusedVariableWarning(_inf5);
+    ignoreUnusedVariableWarning(_inf6);
+    ignoreUnusedVariableWarning(_inf7);
+    ignoreUnusedVariableWarning(_inf8);
+    ignoreUnusedVariableWarning(_pref2);
+    ignoreUnusedVariableWarning(_suf2);
+
```

Comment:
Is there no nicer way to do this? e.g. include the SEQAN_UNUSED macro in the typedef definition?